lib/middleware/request_tracker.rb

```diff
@@ -266,6 +276,14 @@ def rate_limit(env)
     end
   end
 
+  def block_crawler(request)
+    request.get? &&
+      !request.xhr? &&
```

Comment:
I wonder if we should be a bit more atomic in our crawler blocking, but we can defer on this for now and look at numbers first